docs/contributing/development/README.md

```diff
@@ -154,6 +154,32 @@ Finally, should you wish to modify any of the resulting codegen files (ending in
 do not modify them directly. Instead, you can edit the files in
 `cmd/protoc-gen-go-tetragon` and then re-run `make codegen`.
 
+### Docker Container
```

Comment:
How a bout "Building and running a docker image"?

Reply:
Done